### PR TITLE
docs: add 'how to remove route prefix' in FAQ

### LINKED ISSUE #4452: request help:  how apisix remove route  prefix?
### Issue description
for example :
```
origin path is  /apisix/api/*
upstream path should be : /api/*
```
image

how can I use it?

### Environment

Request help without environment information will be ignored or closed.

* apisix version (cmd: `apisix version`):  2.5
* OS (cmd: `uname -a`):
* OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
* etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
* apisix-dashboard version, if have:
* luarocks version, if the issue is about installation (cmd: `luarocks --version`):

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Signed-off-by: yiyiyimu <wosoyoung@gmail.com>

### What this PR does / why we need it:
fix #4452 

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -502,3 +502,37 @@ $ acme.sh --renew --domain demo.domain
 ```
 
 Blog https://juejin.cn/post/6965778290619449351 has detail setup.
+
+## How to remove route prefix for path matching
```
Comment (spacewander) [inline docs/en/latest/FAQ.md:506]: Also, need to update the Chinese version.

Comment (Yiyiyimu) [reply docs/en/latest/FAQ.md:506]: Thanks for reminding! Done

**Review by spacewander — APPROVED** [review]

File: docs/zh/latest/FAQ.md [hunk 2]
```diff
@@ -501,3 +501,37 @@ $ acme.sh --renew --domain demo.domain
 ```
 
 详细步骤，可以参考博客 https://juejin.cn/post/6965778290619449351
+
+## 如何在路径匹配时去除路由前缀
+
+去除路由前缀，比如说从 `/foo/get` 转发至 `/get`，可以通过插件 `proxy-rewrite` 实现。
```
Comment (tzssangglass) [inline docs/zh/latest/FAQ.md:507]: 去除路由前缀，比如说从 `/foo/get` 转发至 `/get`

can be changed to

在转发到上游之前，剪除请求路径中的前缀，比如说从 `/foo/get` 改成 `/get`

Comment (Yiyiyimu) [reply docs/zh/latest/FAQ.md]: Thanks for the suggestion! PTAL

**Review by tzssangglass — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**